controllers/authorino_controller_test.go

```diff
@@ -24,7 +24,7 @@ import (
 const (
 	AuthorinoNamespace = "default"
 	AuthorinoReplicas  = 1
-	AuthorinoImage     = "quay.io/3scale/authorino"
+	AuthorinoImage     = "quay.io/kuadrant/authorino"
```

Comment:
As per comment above we could move it to a const in the authorino_controller and reuse it here

Reply:
Done